cmd/meroxa/root/environments/update.go

```diff
@@ -0,0 +1,228 @@
+/*
+Copyright © 2021 Meroxa Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package environments
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/manifoldco/promptui"
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+var (
+	_ builder.CommandWithDocs    = (*Update)(nil)
+	_ builder.CommandWithArgs    = (*Update)(nil)
+	_ builder.CommandWithFlags   = (*Update)(nil)
+	_ builder.CommandWithClient  = (*Update)(nil)
+	_ builder.CommandWithLogger  = (*Update)(nil)
+	_ builder.CommandWithExecute = (*Update)(nil)
+	_ builder.CommandWithPrompt  = (*Update)(nil)
+)
+
+type updateEnvironmentClient interface {
+	UpdateEnvironment(ctx context.Context, nameOrUUID string, body *meroxa.UpdateEnvironmentInput) (*meroxa.Environment, error)
+}
+
+type Update struct {
+	client updateEnvironmentClient
+	logger log.Logger
+
+	args struct {
+		Name string
```

Comment:
that would be more accurate.  currently, the other environment commands that take an identifier as an arg call it name, though it's passed in to a function that accept name or uuid. 
should we update all of the environment commands now to be identifier instead of name, or would that be out of scope?

connector describe and update but not remove calls it NameOrID. There are some other examples too, but it's not consistent across the board.  I'm not sure if making it consistent everywhere would require changelog type doc updates?